RecoParticleFlow/PFClusterProducer/python/particleFlowCluster_cff.py

```diff
@@ -92,11 +92,91 @@
+from RecoParticleFlow.PFRecHitProducer.hcalRecHitSoAProducer_cfi import hcalRecHitSoAProducer as _hcalRecHitSoAProducer
+from RecoParticleFlow.PFRecHitProducer.pfRecHitHCALParamsESProducer_cfi import pfRecHitHCALParamsESProducer as _pfRecHitHCALParamsESProducer
+from RecoParticleFlow.PFRecHitProducer.pfRecHitHCALTopologyESProducer_cfi import pfRecHitHCALTopologyESProducer as _pfRecHitHCALTopologyESProducer
+from RecoParticleFlow.PFRecHitProducer.pfRecHitSoAProducerHCAL_cfi import pfRecHitSoAProducerHCAL as _pfRecHitSoAProducerHCAL
+from RecoParticleFlow.PFClusterProducer.pfClusterSoAProducer_cfi import pfClusterSoAProducer as _pfClusterSoAProducer
+
+_alpaka_pfClusteringHBHEHFTask = pfClusteringHBHEHFTask.copy() #Full Reco
+_alpaka_pfClusteringHBHEHFOnlyTask = pfClusteringHBHEHFOnlyTask.copy() #HCAL Only
+_alpaka_pfClusteringHBHEHFOnlyLegacyTask = pfClusteringHBHEHFOnlyTask.copy() #HCAL Only + legacy
+
+
+pfRecHitHCALParamsRecordSource = cms.ESSource('EmptyESSource',
+            recordName = cms.string('PFRecHitHCALParamsRecord'),
+            iovIsRunNotTime = cms.bool(True),
+            firstValid = cms.vuint32(1)
+    )
+
+pfRecHitHCALTopologyRecordSource = cms.ESSource('EmptyESSource',
+            recordName = cms.string('PFRecHitHCALTopologyRecord'),
+            iovIsRunNotTime = cms.bool(True),
+            firstValid = cms.vuint32(1)
+    )
+
+pfRecHitHCALParamsESProducer = _pfRecHitHCALParamsESProducer.clone(
+        energyThresholdsHB = cms.vdouble( 0.1, 0.2, 0.3, 0.3 ),
+        energyThresholdsHE = cms.vdouble( 0.1, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2 )
+    )
+
+pfRecHitHCALTopologyESProducer = _pfRecHitHCALTopologyESProducer.clone()
+pfRecHitSoAProducerHCAL = _pfRecHitSoAProducerHCAL.clone(
+        producers = cms.VPSet(
+            cms.PSet(
+                src = cms.InputTag("hbheRecHitProducerPortable"),
+                params = cms.ESInputTag("pfRecHitHCALParamsESProducer:"),
+            )
+        ),
+        topology = "pfRecHitHCALTopologyESProducer:",
+        synchronise = cms.untracked.bool(False)
+    )
+
+
+pfClusterSoAProducer = _pfClusterSoAProducer.clone(
+        pfRecHits = 'pfRecHitSoAProducerHCAL',
+        topology = "pfRecHitHCALTopologyESProducer:",
+        synchronise = cms.bool(False)
+    )
+
+_alpaka_pfClusteringHBHEHFTask.add(pfRecHitHCALParamsRecordSource)
+_alpaka_pfClusteringHBHEHFTask.add(pfRecHitHCALTopologyRecordSource)
+_alpaka_pfClusteringHBHEHFTask.add(pfRecHitHCALParamsESProducer)
+_alpaka_pfClusteringHBHEHFTask.add(pfRecHitHCALTopologyESProducer)
+_alpaka_pfClusteringHBHEHFTask.add(pfRecHitSoAProducerHCAL)
+_alpaka_pfClusteringHBHEHFTask.add(pfClusterSoAProducer)
+
+pfRecHitSoAProducerHBHEOnly = _pfRecHitSoAProducerHCAL.clone(
+        producers = cms.VPSet(
+            cms.PSet(
+                src = cms.InputTag("hbheRecHitProducerPortable"),
+                params = cms.ESInputTag("pfRecHitHCALParamsESProducer:"),
+            )
+        ),
```

Comment:
Could be
```suggestion
        producers = [
            cms.PSet(
                src = cms.InputTag("hbheRecHitProducerPortable"),
                params = cms.ESInputTag("pfRecHitHCALParamsESProducer:"),
            )
        ],
```
or _maybe_ even
```suggestion
        producers = [
            dict(
                src = "hbheRecHitProducerPortable",
                params = "pfRecHitHCALParamsESProducer:",
            )
        ],
```